src/tests/account/test_account.cairo

```diff
@@ -37,6 +41,10 @@ struct SignedTransactionData {
     s: felt252
 }
 
+fn ZERO() -> ContractAddress {
+    contract_address_const::<0>()
+}
+
```

Comment:
let's centralize this as in the [Ownable PR](https://github.com/OpenZeppelin/cairo-contracts/pull/675/)